<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceWithPython</class>
 <widget class="QDialog" name="InterfaceWithPython">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Code</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QToolButton" name="addMatlabCodeBlockToolButton_">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Insert Matlab code block</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Input Variable Names</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="inputVariableNamesTableWidget_">
         <column>
          <property name="text">
           <string>Port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Output Variable Names</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="outputVariableNamesTableWidget_">
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>PythonMatrix1</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Matrix</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>PythonMatrix2</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>Matrix</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>PythonMatrix3</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>Matrix</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>PythonField1</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>Field</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>PythonField2</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>Field</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>PythonField3</string>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>Field</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>PythonString1</string>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string>String</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>PythonString2</string>
          </property>
         </item>
         <item row="7" column="1">
          <property name="text">
           <string>String</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>PythonString3</string>
          </property>
         </item>
         <item row="8" column="1">
          <property name="text">
           <string>String</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Top-level Script</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Execute main now</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Polling interval (ms)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="pollingIntervalSpinBox_">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Retry attempts</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="retryAttemptsSpinBox_">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="enableConsoleOutputCheckBox_">
         <property name="text">
          <string>Enable Console Output</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearObjectPushButton_">
       <property name="text">
        <string>Clear Output Objects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pythonDocPushButton_">
       <property name="text">
        <string>Load Python API Documentation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
